src/pages/support/[secret].tsx

```diff
@@ -91,13 +90,21 @@ const support: NextPage<{ secret: string }> = (props): JSX.Element => {
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 support.getInitialProps = async (ctx: any): Promise<any> => {
-  if (ctx.res && ctx.query.secret !== env.supportSecret) {
-    ctx.res.statusCode = 404;
-    ctx.res.end('Not found');
-  }
+  if(ctx.res){
+    // const { Config } = require ('../../entities/config');
+    // const judgeSecret = await Config.findOne('judgeSecret')
+    // if (ctx.query.secret !== judgeSecret.value) {
+
+    //   ctx.res.statusCode = 404;
+    //   ctx.res.end('Not found');
+    // }
 
+    // return {
+    //   secret: judgeSecret.value
+    // };
+  }
```

Comment:
@johnkahn can you provide feedback on what to do here?

Reply:
Remove them

Reply:
The if statement too, just the whole thing should be deleted